src/app/EventManagement.h

```diff
@@ -566,6 +560,7 @@ class EventManagement
     Messaging::ExchangeManager * mpExchangeMgr = nullptr;
     EventManagementStates mState               = EventManagementStates::Shutdown;
     uint32_t mBytesWritten                     = 0;
+    static System::Mutex mEventLock;
```

Comment:
When reading the header, I am wondering what 'event lock' protects. I would have expected a 'Event static member' but that is not the case:

- Why is this static?
- can we name it like "EventBufferLock" if it is supposed to the event buffer?

Reply:
just remove static, previously there is check static_assert(std::is_trivially_destructible<EventManagement>::value, "EventManagement must be trivially destructible"); Currently System::Mutex is not trivally destructible, I think it is ok to remove this static_assert